packages/dialect-react-ui/components/Chat/screens/Main/pages/ThreadPage/MessageInput.tsx

```diff
@@ -40,7 +40,7 @@ export default function MessageInput({
         <form onSubmit={onSubmit}>
           <div className="dt-relative">
             <div className="dt-text-sm dt-break-words dt-py-1 dt-pl-2 dt-pr-11">
-              {text || 'h'}
+              {text.trim() || 'h'}
```

Comment:
I think it might be better to have text trimmed in state, but not only in visualization. otherwise it might create misleading to user. e.g. it shows empty input box, but in reality it contains symbols that got trimmed

Reply:
The text gets rendered in the `<Textarea ... />` below on line 47. So what the user sees is what they send in the protocol. I believe this is just a placeholder to ensure the div doesn't collapse.

Reply:
ok, got it